lib/taskana-core/src/test/java/acceptance/config/TaskanaSecurityConfigAccTest.java

```diff
@@ -111,10 +112,27 @@ private void setSecurityFlag(boolean securityFlag) throws Exception {
 
     try (Connection connection = DataSourceGenerator.getDataSource().getConnection()) {
 
-      String sql =
-          String.format(
-              "UPDATE %s.CONFIGURATION SET ENFORCE_SECURITY = %b WHERE NAME = 'MASTER'",
-              DataSourceGenerator.getSchemaName(), securityFlag);
+      String dbProductId =
+          DB.getDatabaseProductId(connection.getMetaData().getDatabaseProductName());
+      String sql;
+      if (DB.isOracle(dbProductId)) {
+        if (securityFlag) {
+          sql =
+              String.format(
+                  "UPDATE %s.CONFIGURATION SET ENFORCE_SECURITY = 1 WHERE NAME = 'MASTER'",
+                  DataSourceGenerator.getSchemaName());
+        } else {
+          sql =
+              String.format(
+                  "UPDATE %s.CONFIGURATION SET ENFORCE_SECURITY = 0 WHERE NAME = 'MASTER'",
+                  DataSourceGenerator.getSchemaName());
+        }
+      } else {
+        sql =
+            String.format(
+                "UPDATE %s.CONFIGURATION SET ENFORCE_SECURITY = %b WHERE NAME = 'MASTER'",
+                DataSourceGenerator.getSchemaName(), securityFlag);
+      }
```

Comment:
better: convert `securityFlag` to a numeric value for oracle?

Reply:
oracle don't have boolean types
0 is false, 1  is true

I don't get your point: 
> convert `securityFlag` to a numeric value for oracle?